apps/admin-gui/src/app/shared/components/services-list/services-list.component.ts

```diff
@@ -92,6 +92,21 @@ export class ServicesListComponent implements AfterViewInit, OnChanges {
     );
   }
 
+  exportDisplayedData(format: string): void {
+    const start = this.dataSource.paginator.pageIndex * this.dataSource.paginator.pageSize;
+    const end = start + this.dataSource.paginator.pageSize;
+    downloadData(
+      getDataForExport(
+        this.dataSource
+          .sortData(this.dataSource.filteredData, this.dataSource.sort)
+          .slice(start, end),
+        this.displayedColumns,
+        ServicesListComponent.getDataForColumn as (data: Service, column: string) => string
```

Comment:
Same as before. Is this retype needed?

Reply:
Same here.

Reply:
Sorry, my bad. The comment was meant to say if we could remove this explicit typing without any repercussions.